modules/log.cpp

```diff
@@ -457,12 +457,26 @@ CModule::EModRet CLogMod::OnSendToIRCMessage(CMessage& Message) {
     return CONTINUE;
 }
 
-void CLogMod::OnJoin(const CNick& Nick, CChan& Channel) {
-    if (NeedJoins()) {
-        PutLog("*** Joins: " + Nick.GetNick() + " (" + Nick.GetIdent() + "@" +
-                   Nick.GetHost() + ")",
-               Channel);
+void CLogMod::OnJoinMessage(CJoinMessage& Message) {
+    if (!NeedJoins())
+        return;
+
+    const CNick& Nick = Message.GetNick();
+    CChan& Channel = *Message.GetChan();
+    CString Account = Message.GetTag("account");
```

Comment:
please add `s` prefix to the string for consistency with the rest: sAccount

Reply:
Okay, I'll push that out soon.

Reply:
I think I may have misunderstood, I put the space as a separate string to avoid the extra overhead of having to prepend a space onto the Account string.  I can certainly do that but I just felt this was a bit more efficient this way.

Reply:
You have misunderstood. I know why you made `s` variable separately. I meant `sAccount` literally. All CString variables here start with symbol `s`

Reply:
Oh, that's actually what I originally thought.  Okay I'll change that.

Reply:
Done, it builds but I'm still testing.  I don't anticipate any issues with a one letter change to a variable name, though.

Reply:
Of course

Reply:
Tested with the variable name change and still works.